astrospec/spectrum1d.py

```diff
@@ -1,6 +1,7 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
+
 import numpy as np
-from scipy inport interpolate
+from scipy import interpolate
 
```

Comment:
astropy does not have scipy as a dependency, but I think we can do that. Do you agree?